app/server/controllers/UserController.js

```diff
@@ -183,6 +183,45 @@ UserController.createUser = function (email, password, callback) {
   });
 };
 
+/**
+ * Create a new walkin user given an email and use information.
+ * @param  {String}   req    User's information.
+ * @param  {Function} callback args(err, user)
+ */
+UserController.createWalkinUser = function (req, callback) {
```

Comment:
don't give req. as param. Instead, pass a user obj. Keep funcs small and specific.